libnetwork/osl/namespace_linux.go

```diff
@@ -692,5 +700,8 @@ func (n *networkNamespace) ApplyOSTweaks(types []SandboxType) {
 		case SandboxTypeLoadBalancer:
 			kernel.ApplyOSTweaks(loadBalancerConfig)
 		}
```

Comment:
Looks like this curly brace shouldn't be there (both should be within the switch?)